extensions/wikia/Chat2/templates/Chat_Index.php

```diff
@@ -108,8 +108,8 @@
 					<?= DesignSystemHelper::renderSvg('wds-avatar-badges-discussion-moderator'); ?>
 				<% } else if (groups.indexOf('helper') !== -1) { %>
 					<?= DesignSystemHelper::renderSvg('wds-avatar-badges-helper', 'wds-icon wds-icon-small'); ?>
-				<% } else if (groups.indexOf('vstf') !== -1) { %>
-					<?= DesignSystemHelper::renderSvg('wds-avatar-badges-vstf', 'wds-icon wds-icon-small'); ?>
+				<% } else if (groups.indexOf('soap') !== -1) { %>
+					<?= DesignSystemHelper::renderSvg('wds-avatar-badges-soap', 'wds-icon wds-icon-small'); ?>
```

Comment:
Just question: Do we know if this badges are created already?

Reply:
I'll confirm with Iwing